README.md

```diff
@@ -1,80 +1,92 @@
 
+![Screenshot](https://cdn.jsdelivr.net/gh/yoobool/flask-state@1.0.5/examples/static/flask_state.png)
 
 ## Documentation
-To check out [live example](https://flask-state.herokuapp.com/), and visit [tutorials doc](https://github.com/yoobool/flask-state/wiki/Tutorials).
 
+Check out the [live demo](https://flask-state.herokuapp.com/), or head over to the
+[tutorial](https://github.com/yoobool/flask-state/wiki/Tutorials) for more instructions.
 
 ## Installation
-Install and update using [pip](https://pip.pypa.io/en/stable/quickstart/):
-```
-$ pip install Flask-State
+
+Get this plugin from [PyPI](https://pip.pypa.io/en/stable/quickstart/):
+
+```bash
+pip install Flask-State
 ```
 
-Display components can use ```<script>``` tag from a CDN, or as a flask-state package on npm.
+Alternatively, install Flask-State via NPM or include this script tag to the head
+section of your HTML document:
+
 ```html
 <script src="https://cdn.jsdelivr.net/gh/yoobool/flask-state@v1.0.5/packages/umd/flask-state.min.js"></script>
 ```
-```
+
+```bash
 npm install flask-state --save
 ```
 
-
 ## Usage
 
-### Firstly：Bind database address.
+### 1. Bind Database Address
+
 ```python
 from flask_state import DEFAULT_BIND_SQLITE
 app.config['SQLALCHEMY_BINDS'] = {DEFAULT_BIND_SQLITE: 'sqlite:///path'}
 ```
 
-### Secondly：Call the init_app method of the flask-state to initialize the configuration.
+### 2. Call `init_app` to configure Flask-State
+
 ```python
 import flask_state
 flask_state.init_app(app)
 ```
 
-### Thirdly：Import the view file.
+### 3. Include Imports to Views
+
 ```javascript
-// npm
-// Need to introduce Echarts module
+// requires echarts module
 import 'echarts';
 import 'flask-state/flask-state.min.css';
 import {init} from 'flask-state';
-// Create a DOM node with ID 'test'. After init() binds the node, click to open the listening window
+// Create a DOM node with ID 'test'. After init() binds the node, 
+// click to open the listening window
 init({dom:document.getElementById('test')});
 ```
 
-**Learn more about advanced configurations by reading** [documentation](https://github.com/yoobool/flask-state/wiki/Configuration).
+**Learn more about advanced configurations in the**
+[documentation](https://github.com/yoobool/flask-state/wiki/Configuration).
 
 ## Contributing
-Welcome to [open an issue](https://github.com/yoobool/flask-state/issues/new)!
 
-Flask-State follows the [Contributor Covenant](https://www.contributor-covenant.org/version/1/3/0/code-of-conduct/) Code of Conduct.
+You're welcome to [open a new issue](https://github.com/yoobool/flask-state/issues/new)!
+Flask-State follows the [Contributor Covenant](https://www.contributor-covenant.org/version/1/3/0/code-of-conduct/)
+Code of Conduct.
 
 ## Community Channel
-We're on [Gitter](https://gitter.im/flaskstate/community) ! Please join us.
+
+We're on [Gitter](https://gitter.im/flaskstate/community)! Join the conversation
+fore more questions and inquiries about this project.
```

Comment:
**Correct:** fore -> for more questions and inquiries about this project.

Reply:
Thanks for your suggestions, I'm on it!